File: src/CitiesOnMap.WebUI/src/app/_models/game/gameModel.ts
import {GameOptionsModel} from "./gameOptionsModel";

export interface GameModel {
  id: string;
  playerId: string;
  points: number;
  currentCityName: string | null;
  cityPopulation: number | null;
  country: string | null;
  lastPlayTime: Date;
  options: GameOptionsModel;
}

File: src/CitiesOnMap.Application/Features/Games/Models/GameModel.cs
namespace CitiesOnMap.Application.Features.Games.Models;

public class GameModel
{
    public string Id { get; set; } = null!;
    public string PlayerId { get; set; } = null!;
    public int Points { get; set; }
    public string? CurrentCityName { get; set; }
    public int? CityPopulation { get; set; }
    public string? Country { get; set; }
    public DateTimeOffset LastPlayTime { get; set; }
    public GameOptionsModel Options { get; set; } = null!;
}
